src/uct/ib/base/ib_iface.c

```diff
@@ -205,6 +205,22 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "Reverse Service level. 'auto' will set the same value of sl\n",
    ucs_offsetof(uct_ib_iface_config_t, reverse_sl), UCS_CONFIG_TYPE_ULUNITS},
 
+  {"SEND_OVERHEAD", UCS_VALUE_AUTO_STR,
```

Comment:
Would it make sense to add unit tests for these changes?

Reply:
Hmm... I'm not sure that unit testing makes sense here. I haven't changed the behavior, but added configuration parameters... The only thing that comes to mind is to add test that calls `uct_iface_estimate_perf`, and checks the values. But we already have such test. Thus, it will not increase code coverage. Do you mean smth like this?

Reply:
I was thinking of the test that does 3 things:
1) configures parameters with newly provided options
2) Runs some business logic that calls `uct_iface_estimate_perf`, that should use new config
3) Asserts that it behaves as expected

But up to you to decide, I've no idea about complexity of such a test

Reply:
If you don't mind I will add the following test in the separate PR right after merging this PR. Because I don't want to trigger once again the CI for this PR.
<details>
  <summary>TEST</summary>
    
```cpp
    UCS_TEST_P(test_ust_ib_iface, estimate_perf,
               "IB_SEND_OVERHEAD=bcopy:1,cqe:2,db:3,wqe_fetch:4,wqe_post:5")
    {
        uct_perf_attr_t perf_attr = {
            .field_mask         = UCT_PERF_ATTR_FIELD_OPERATION |
                                  UCT_PERF_ATTR_FIELD_LOCAL_MEMORY_TYPE |
                                  UCT_PERF_ATTR_FIELD_REMOTE_MEMORY_TYPE |
                                  UCT_PERF_ATTR_FIELD_LOCAL_SYS_DEVICE |
                                  UCT_PERF_ATTR_FIELD_REMOTE_SYS_DEVICE |
                                  UCT_PERF_ATTR_FIELD_SEND_PRE_OVERHEAD |
                                  UCT_PERF_ATTR_FIELD_SEND_POST_OVERHEAD |
                                  UCT_PERF_ATTR_FIELD_LATENCY,
            .operation          = UCT_EP_OP_LAST,
            .local_memory_type  = UCS_MEMORY_TYPE_HOST,
            .remote_memory_type = UCS_MEMORY_TYPE_HOST,
            .local_sys_device   = UCS_SYS_DEVICE_ID_UNKNOWN,
            .remote_sys_device  = UCS_SYS_DEVICE_ID_UNKNOWN
        };
        for (auto op = UCT_EP_OP_AM_SHORT; op < UCT_EP_OP_LAST;
             op = uct_ep_operation_t(op + 1)) {
            perf_attr.operation = op;
            EXPECT_EQ(estimate_perf(&perf_attr), UCS_OK);
            EXPECT_FLOAT_EQ(perf_attr.send_post_overhead,
                            uct_ep_op_is_zcopy(op) ? 5. : 3.);
            EXPECT_FLOAT_EQ(perf_attr.send_pre_overhead,
                            uct_ep_op_is_bcopy(op) ? 6. : 5.);
            EXPECT_FLOAT_EQ(perf_attr.latency.c,
                            (uct_ep_op_is_bcopy(op) || uct_ep_op_is_zcopy(op)) ?
                                (attr_latency_c() + 4.) : attr_latency_c());
        }
    }
```

</details>

Reply:
thanks!